src/main/java/org/example/App.java

```diff
@@ -32,10 +38,6 @@ public static void main( String[] args ) {
 
 
         List<Route> allRoutes = routeService.getRoutes();
```

Comment:
add System.out.println("Loading cities..."); since it takes some time to load them